docs/install/install.md

```diff
@@ -19,30 +19,173 @@ limitations under the License.
 
-### Build from Zeppelin Source
+####1. Clone Zeppelin repository
 
-You can also build Zeppelin from the source.
+```
+git clone https://github.com/apache/zeppelin.git
+```
+
+####2. Build source with options 
+Each interpreters requires different build options. For the further information about options, please see [Build](https://github.com/apache/zeppelin#build) section.
+
+```
+mvn clean package -DskipTests [Options]
+```
+
+Here are some examples with several options
+
+```
+# basic build
+mvn clean package -Pspark-1.6 -Phadoop-2.4 -Pyarn -Ppyspark
 
-#### Prerequisites for build
- * Java 1.7
- * Git
- * Maven(3.1.x or higher)
- * Node.js Package Manager
+# spark-cassandra integration
+mvn clean package -Pcassandra-spark-1.5 -Dhadoop.version=2.6.0 -Phadoop-2.6 -DskipTests
 
-If you don't have requirements prepared, please check instructions in [README.md](https://github.com/apache/zeppelin/blob/master/README.md) for the details.
+# with CDH
+mvn clean package -Pspark-1.5 -Dhadoop.version=2.6.0-cdh5.5.0 -Phadoop-2.6 -Pvendor-repo -DskipTests
+
+# with MapR
+mvn clean package -Pspark-1.5 -Pmapr50 -DskipTests
+```
+
+For the further information about building with source, please see [README.md](https://github.com/apache/zeppelin/blob/master/README.md) in Zeppelin repository.
+
+## Starting Zeppelin with Command Line
+#### Start Zeppelin
+
+```
+bin/zeppelin-daemon.sh start
+```
+
+If you are using Windows 
+
+```
+bin\zeppelin.cmd
+```
+
+After successful start, visit [http://localhost:8080](http://localhost:8080) with your web browser.
+
+#### Stop Zeppelin
+
+```
+bin/zeppelin-daemon.sh stop
+```
+
+#### (Optional) Start Zeppelin with a service manager
+
+> **Note :** The below description was written based on Ubuntu Linux.
+
+Zeppelin can be auto started as a service with an init script, such as services managed by **upstart**.
+
+The following is an example of upstart script to be saved as `/etc/init/zeppelin.conf`
+This also allows the service to be managed with commands such as
+
+```
+sudo service zeppelin start  
+sudo service zeppelin stop  
+sudo service zeppelin restart
+```
+
+Other service managers could use a similar approach with the `upstart` argument passed to the `zeppelin-daemon.sh` script.
+
+```
+bin/zeppelin-daemon.sh upstart
+```
+
+**zeppelin.conf**
+
+```
+description "zeppelin"
+
+start on (local-filesystems and net-device-up IFACE!=lo)
+stop on shutdown
+
+# Respawn the process on unexpected termination
+respawn
+
+# respawn the job up to 7 times within a 5 second period.
+# If the job exceeds these values, it will be stopped and marked as failed.
+respawn limit 7 5
+
+# zeppelin was installed in /usr/share/zeppelin in this example
+chdir /usr/share/zeppelin
+exec bin/zeppelin-daemon.sh upstart
+```
+
+## What is the next?
+Congratulation to your successful Zeppelin installation! Here are two next steps you might need.
```

Comment:
I belive it should be `Congratulation _ob_` as it's not an Apache Zeppelin installation that is being congratulated here, but the user.